app/javascript/components/region-form/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import createSchema from './region-form.schema';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import { API } from '../../http_api';
+import miqRedirectBack from "../../helpers/miq-redirect-back";
+
+const RegionForm = ({ id, maxDescLen }) => {
+  const [initialValues, setInitialValues] = useState({});
+  const [isLoading, setIsLoading] = useState(!!id);
+
+  useEffect(() => {
+    if (id) {
+      miqSparkleOn();
+      API.get(`/api/regions/${id}?attributes=description`)
+        .then(({
+                 ...data
```

Comment:
You can just simply use `data` without the `{...}`

Reply:
Thinking this through, maybe you just need the `description` field and nothing more, so you don't even need the whole `initialValues`, just a `{ description }`